### DISCUSSION TITLE
如何禁用transport.enableClientBatchSendRequest

### DISCUSSION THREAD

**Discussion by will-zdu** [apache/incubator-seata#6720] [General]
当前是模式file，我尝试了几次，找不到对应的文档，从代码也看不出来

**Comment by funky-eyes** [comment]
为什么要禁用该配置？1.6上数组越界问题已经被1.6.1及以上版本修复。如果你想关闭，请把这个配置配置到你设置给seata-server的配置中心中
Why should this configuration be disabled? The array out-of-bounds issue on 1.6 has already been fixed in versions 1.6.1 and above. If you wish to disable it, please configure this setting in the configuration center provided to the Seata server

**Comment by will-zdu** [comment]
因为做分布式调用链路追踪的时候发现，io.seata.core.rpc.netty.AbstractNettyRemotingClient#sendSyncRequest(java.lang.Object)
这个方法开启了NettyClientConfig.isEnableClientBatchSendRequest()，会将rpcMessage合并发送，合并发送也没有什么问题，问题是为什么合并发送的时候
` MergedWarpMessage mergeMessage = new MergedWarpMessage();
                    while (!basket.isEmpty()) {
                        RpcMessage msg = basket.poll();
                        mergeMessage.msgs.add((AbstractMessage) msg.getBody());
                        mergeMessage.msgIds.add(msg.getId());
                    }`
                    
拷贝消息为什么把消息头没有给拷贝，做分布式链路追踪的时候就很麻烦了

禁用的问题我已经解决了，因为springboot里面配置文件优先级更高，改了springboot配置文件优先级就解决了

**Reply by funky-eyes** [reply]
是的，这一直是一个问题
Yes, this has always been a problem
